### DISCUSSION TITLE
Wiki practically dying on its own

### DISCUSSION THREAD

**Discussion by Fr3shlama** [requarks/wiki#6356] [Help / Questions]
Im having a weird issue with my wikijs instance running on https://wiki.die-hills.at

Basically: My Wiki was running fine for a year now, when it suddenly decided to die. Im using the docker setup
The database seems to be working fine and the wiki does not seem to report any errors

But if I go to my admin panel I see this:

![grafik](https://user-images.githubusercontent.com/25925470/231836374-3accca2d-4536-4238-81b4-98a52c4bbfdf.png)
![grafik](https://user-images.githubusercontent.com/25925470/231836882-05a26f17-ae9c-4a21-bb1b-08a802abb934.png)

Nothing is loading

![grafik](https://user-images.githubusercontent.com/25925470/231837022-1388e5c8-2cb1-471f-bc36-4d7896c5f283.png)
Docker logs seem fine also

Im running a db container, a wiki container and a wiki-update container

Last time I had that error I rebuilt the wiki and wiki-update container but it doesnt seem to work this time around

**Comment by bloodrizer** [comment]
Same issue here

**Comment by NGPixel** [comment]
Are you using a reverse-proxy like Cloudflare? Try to enable development mode on it. If it works afterward, it means some rule at cloudflare is blocking JS / api calls.

**Comment by abhishekgupta0912** [comment]
Hey I am also facing the same issue.. 
Any leads on above.

**Reply by Fr3shlama** [reply]
I reinstalled the wiki without the docker containers and on mariadb and it seems to work fine now

I do use cloudflare but I have all scripts and apps disabled and only use it to hide the server IP

**Reply by abhishekgupta0912** [reply]
i am not running it as a container rather i created a deployment for it and then using it. It was working fine but form past 4 days i am not able to click on the buttons and the tab groups that are displayed on the web page of wiki. If I do kubectl port-forward in that case it is working as expected. if it is the issue with the cloud flare i tried enabling developers modes but it was still not working.
